object-cache.php

```diff
@@ -762,6 +771,11 @@ protected function _call_redis( $method ) {
 			}
 		}
 
+		if ( $this->is_redis_wakeup_flush_enabled() && ! $this->redis_needs_flush ) {
```

Comment:
Given than naming is one of the two hard problems, can I request we call this feature the `redis_failback_flush`?

The term fail-back is a generally used thing in HA systems, and it's widely accepted that "failing back" to the nominally primary system is as risky and complex as the initial fail-over. That's the frame of reference I'd like to promote in thinking about this feature.